lib/node_modules/@stdlib/math/base/special/sincospi/lib/main.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+// MODULES //
+
+var computeSincospi = require( './sincospi.js' );
```

Comment:
This is a bit of an ugly variable name. Simplify to `compute` or even just `fcn`, as its meaning can be inferred from the filename `sincospi.js`.

Reply:
I think there is precedent for using `compute` in such cases inside of `stdlib`. `fcn` not so much.